modules/profiling-ext/README.txt

```diff
@@ -0,0 +1,12 @@
+Apache Ignite Profiling Module
+------------------------------
+
+Apache Ignite Profiling module provides cluster profiling tool based on performance logging to profiling files.
+
+To collect statistics in runtime and to build the performance report follow:
+
+ - Start profiling (use IgniteProfilingMBean JMX bean).
+ - Collect workload statistics.
+ - Stop profiling (use IgniteProfilingMBean JMX bean).
+ - Collect profiling files from all nodes under an empty directory.
+ - Run script ./bin/profiling.sh path_to_files to build the performance report.
```

Comment:
Can you please share more info on the usage of this tool?

 In the readme we have ./bin/profiling.sh but this file is not present in PR.

Reply:
I have updated PR and readme file. Thanks for taking a look.